tests/exceptions/exceptions_errc_for.cpp

```diff
@@ -0,0 +1,89 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Provides tests for sycl::errc_for
+//
+*******************************************************************************/
+
+#include "exceptions.h"
+
+#define TEST_NAME exceptions_errc_for
+
+namespace TEST_NAMESPACE {
+
+template <template <sycl::backend> class arg>
+struct check_template_exists {};
+
+bool check_opencl_supporting(const sycl::queue &q,
+                             sycl_cts::util::logger &log) {
+  bool opencl_supported{};
```

Comment:
```suggestion
  bool opencl_supported{false};
```

Reply:
I've applied this comment in this commit: 4cc40890ad0cab86cb55479a71fe92b096c2be18